src/jobflow_remote/cli/job.py

```diff
@@ -211,40 +208,63 @@ def jobs_list(
                 name=name,
                 metadata=metadata,
                 workers=worker_name,
-                limit=max_results,
-                sort=db_sort,
             )
+        out_console.print(f"Selected jobs: {n_jobs}")
```

Comment:
Same as above for the Flows, I don't know if it is more clear or not and I let you decide :)
```suggestion
        out_console.print(f"Number of jobs matching query: {n_jobs}")
```